src/App.js

```diff
@@ -1,38 +1,19 @@
-import React, { Component } from 'react';
+import React from 'react';
+import MapForm from './components/MapForm'
 import './App.css';
-import Chart from './Chart';
+import Chart from './components/Chart';
 
-class App extends Component {
-  state = {
-    APIData: [],
-    fromYear: 2018
-  }
-
-  handleAPIClick = () => {
-    fetch('https://api.census.gov/data/timeseries/eits/resconst?get=cell_value,data_type_code,time_slot_id,error_data,category_code,seasonally_adj&time=from+' + this.state.fromYear)
-        .then(res => res.json())
-        .then((data) => {
-          this.setState({
-            APIData: data
-          });
-          console.log(this.state.APIData);
-        })
-        .catch(console.log)
-  };
-
-  render() {
-    return (
-      <div className="App">
-        <button onClick={this.handleAPIClick}>Click to test API</button>
-        <div>
-          <h2>Data in Console</h2>
-          <p>Data is stored in the state as APIData</p>
-        </div>
+function App() {
+  return (
+    <div className="App">
+      <div>
+        <MapForm />
+      </div>
+      <div>
         <Chart />
```

Comment:
Should we build the chart component as a child component of MapForm? Seems like that'd be the easiest way to pass it the data, we could have `<Chart labels=labels data=data />` then it would auto-reload whenever the form's state changes too.

Reply:
Also I think this would resolve the overlapping issue, if `<chart ... />` was inside of `<Map ... />`

Reply:
That makes sense to me. I just left the two components there for now to be figured out soon. I'm not sure it will solve the overlapping issue. Seems weird that it's happening as is.